src/stories/utils/show-more.js

```diff
@@ -0,0 +1,36 @@
+document.addEventListener("DOMContentLoaded", () => {
+  const categories = document.querySelectorAll("[data-show-more-item]");
+  const toggleButton = document.querySelector("[data-show-more-button]");
+  const initialVisibleItems =
+    document.querySelector("[data-show-more-initial-visible-items]") ?? 2; // default to 2 if not set
+
+  // if there are no categories or only one category, hide the toggle button
+  // or if there are less categories than the initial visible items, hide the toggle button
+  if (categories.length <= 1 || categories.length <= initialVisibleItems) {
+    toggleButton.style.display = "none";
```

Comment:
I'd really prefer you use a class instead, e.g. `.is-hidden` or something.
I dont really want inline styles when we can avoid it.